functions/volunteer.inc

```diff
@@ -602,8 +602,12 @@ SQL;
 }
 
 
-function volunteers_in_dept($department)
+function volunteers_in_dept($department, $event = null)
 {
+    if ($event === null) {
+        $event = current_eventID();
```

Comment:
In PHP, can the initializer in the argument list be a method call? That is, can you just say

```php
function volunteers_in_dept($department, $event = current_eventID())
```

and skip a step?

Reply:
I do not believe so. Or at least a few minutes of web searching did not turn up a way to do that. And quick test programs do not work.

Reply:
👍